server/src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -512,7 +519,9 @@ public Consumer getHost(String guestId, Owner owner) {
             .addOrder(Order.desc("updated"))
             .setMaxResults(1)
             .setProjection(Projections.property("consumer"));
-        return (Consumer) crit.add(guestIdCrit).uniqueResult();
+        Consumer host = (Consumer) crit.add(guestIdCrit).uniqueResult();
```

Comment:
Can we move the .add to the block above where we're adding all the other restrictions? 

```
         Criteria crit = currentSession()
             .createCriteria(GuestId.class)
             .createAlias("consumer", "gconsumer")
             .add(Restrictions.eq("gconsumer.owner", owner))
             .add(guestIdCrit)
             .addOrder(Order.desc("updated"))
             .setMaxResults(1)
             .setProjection(Projections.property("consumer"));

        Consumer host = (Consumer) crit.uniqueResult();
```